Mathlib/Probability/Distributions/Uniform.lean

```diff
@@ -172,9 +224,34 @@ theorem integral_eq (huX : IsUniform X s ℙ) :
 
 end IsUniform
 
-end pdf
+variable {X : Ω → E}
+
+lemma uniformMeasure.IsUniform {s : Set E} : IsUniform (id : E → E) s (uniformMeasure s μ) μ := by
+  unfold IsUniform
+  rw [Measure.map_id]
+
+/-- The density of the uniform measure on a set with respect to itself. This allows us to abstract
+away the choice of random variable and probability space. -/
+def uniformPDF {s : Set E} {x : E} : ℝ≥0∞ := s.indicator ((μ s)⁻¹ • (1 : E → ℝ≥0∞)) x
+
+/-- Alternative definition for `uniformPDF` based on if-then-else rather than Set.indicator. -/
+def uniformPDF_ite {s : Set E} {x : E} : ℝ≥0∞ := if x ∈ s then (μ s)⁻¹ else 0
```

Comment:
Having two definitions like this is a bad idea. You should keep the first one, and add a lemma saying it expands to `if x ∈ s then (μ s)⁻¹ else 0`

Reply:
I've removed it, thank you!